blocks/HARDWARE/PROTOCOLS/CAN/BUS/PEAK_DETECT_AVAILABLE_DEVICES/PEAK_DETECT_AVAILABLE_DEVICES.py

```diff
@@ -0,0 +1,38 @@
+from flojoy import flojoy, String, DataContainer
+import can
+from typing import Optional
+
+
+@flojoy(deps={"python-can": "4.3.1", "uptime": "3.0.1"})
+def PEAK_DETECT_AVAILABLE_DEVICES(default: Optional[DataContainer]) -> String:
+    """Detects available PCAN devices and returns a string containing the device name and the channel number.
+
+    This blocks is use to detect available PCAN devices and returns a string containing the device name and the channel number.
+    → Use the channel number to configure to connect to PEAK can hardware and other devices that support the PCAN interface using the PCAN_CONNECT block.
+
+    For Windows and Linux compatibilities, the following Drviers are required:
+        → https://www.peak-system.com/Drivers.523.0.html?&L=1
+    For MacOS compatibilities, the following Drivers are required:
+        → https://mac-can.github.io/
+
+    Parameters
```

Comment:
I believe when params are none, we just remove this section.

Reply:
Just tried it out; the docs generator requires it :)